.travis.yml

```diff
@@ -18,4 +17,5 @@ addons:
       - ubuntu-toolchain-r-test
     packages:
       - g++-4.8
-      - oracle-java8-set-default
+before_install:
```

Comment:
Do we still need this on node 8?

Reply:
Oh, which version started shipping npm6?

Reply:
looks like 8.12.0; will fix.